core/system/inc/page_allocator.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __PAGE_ALLOCATOR_H__
+#define __PAGE_ALLOCATOR_H__
+
+#include "api/inc/page_allocator_exports.h"
+
```

Comment:
Could you add quick doxygen docs to these APIs? Like [here](https://github.com/ARMmbed/uvisor/pull/221/files#diff-3208d6868dff7617284efcbbbc7ac9f8R57). Even if they are internal to uVisor it's useful for us. Concentrate on warnings/assumptions if you want, it will help verification.